### PR TITLE
Fixed package.json to include keywords

### PR DESCRIPTION
**Pull request by raygervais**
I noticed that you did not have keywords for this module, so I added ones that seemed relevant. If you'd like others, or different ones, I'd be happy to add them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,19 @@
...
     "node": "*"
   },
+  "keywords" : [
+    "wildcard",
+    "regex",
+    "patterns",
+    "finding",
+    "filesystem",
+    "directories",
+    "utilitly",
```
Comment (shinnn) [inline package.json]: utilit**i**y

**Comment by raygervais** [comment]
@shinnn, thank you for the catch. I'm impressed that my editor did not warn me about it. Regardless, fixed.

**Comment by raygervais** [comment]
@shinnn, when you get a moment could you look over the PR again? It would be advantageous to your module to have keywords which help make the package discoverable.

**Comment by stamminator** [comment]
Not sure why a PR as simple as this wouldn't have been merged a long time ago.

**Comment by raygervais** [comment]
Agreed. This was one of my first attempts at open source contributing, rather disheartening that something so simple was never looked at after the  first glance.

**Pull request closed without merging**